Gemfile

```diff
@@ -5,9 +5,9 @@ platforms :ruby do
 end
 
 group :test do
-  gem 'activerecord', (ENV['RAILS_VERSION'] || ['>= 3.0', '< 4.2'])
+  gem 'activerecord', (ENV['RAILS_VERSION'] || ['>= 3.0', '< 5.0'])
```

Comment:
So the thing is, it's a test dependency. If at runtime the gem relies on AR then this should be a runtime dependency.

If it is indeed required at runtime then to ensure it works with the various Rails versions we use in the company you should use Appraisal, like for instance MW does ([Appraisal file](https://github.com/mdsol/monitor_wares/blob/develop/Appraisals), [travis conf](https://github.com/mdsol/monitor_wares/blob/develop/.travis.yml). I assume this RAILS_VERSION env var is a way to pass the version so one can try to do this but this solution is very brittle and the travis CI file is not even making use of it. If you look at the jobs, they all pulled rails 4.2.
As it stands now people are being misled into thinking this gem is tested with different versions when in reality unless that env var is set and the bundle re-done _each time_ it will only ever test a single version.